### ISSUE TITLE
Error while running `streamlit run promptsource/promptsource.py`

### ISSUE THREAD

**Issue by tae898** [bigscience-workshop/promptsource#520]
It's been a while since I ran `streamlit run promptsource/promptsource.py`.
As always I fetched the latest main branch and ran the command, which threw an error below:

```bash
021-10-08 20:47:11.620 Couldn't find file locally at jigsaw_unintended_bias/jigsaw_unintended_bias.py, or remotely at https://raw.githubusercontent.com/huggingface/datasets/1.7.0/datasets/jigsaw_unintended_bias/jigsaw_unintended_bias.py.
The file was picked from the master branch on github instead at https://raw.githubusercontent.com/huggingface/datasets/master/datasets/jigsaw_unintended_bias/jigsaw_unintended_bias.py.
2021-10-08 20:47:16.489 multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/home/tk/.virtualenvs/promptsource/lib/python3.8/site-packages/datasets/load.py", line 320, in prepare_module
    local_path = cached_path(file_path, download_config=download_config)
  File "/home/tk/.virtualenvs/promptsource/lib/python3.8/site-packages/datasets/utils/file_utils.py", line 281, in cached_path
    output_path = get_from_cache(
  File "/home/tk/.virtualenvs/promptsource/lib/python3.8/site-packages/datasets/utils/file_utils.py", line 621, in get_from_cache
    raise FileNotFoundError("Couldn't find file at {}".format(url))
FileNotFoundError: Couldn't find file at https://raw.githubusercontent.com/huggingface/datasets/1.7.0/datasets/story_cloze/story_cloze.py

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/tk/.virtualenvs/promptsource/lib/python3.8/site-packages/datasets/load.py", line 332, in prepare_module
    local_path = cached_path(file_path, download_config=download_config)
  File "/home/tk/.virtualenvs/promptsource/lib/python3.8/site-packages/datasets/utils/file_utils.py", line 281, in cached_path
    output_path = get_from_cache(
  File "/home/tk/.virtualenvs/promptsource/lib/python3.8/site-packages/datasets/utils/file_utils.py", line 621, in get_from_cache
    raise FileNotFoundError("Couldn't find file at {}".format(url))
FileNotFoundError: Couldn't find file at https://raw.githubusercontent.com/huggingface/datasets/master/datasets/story_cloze/story_cloze.py

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 48, in mapstar
    return list(map(*args))
  File "/home/tk/repos/promptsource/promptsource/promptsource.py", line 109, in get_infos
    all_infos[d_name] = get_dataset_infos(d_name)
  File "/home/tk/.virtualenvs/promptsource/lib/python3.8/site-packages/datasets/inspect.py", line 110, in get_dataset_infos
    module_path, _ = prepare_module(path)
  File "/home/tk/.virtualenvs/promptsource/lib/python3.8/site-packages/datasets/load.py", line 340, in prepare_module
    raise FileNotFoundError(
FileNotFoundError: Couldn't find file locally at story_cloze/story_cloze.py, or remotely at https://raw.githubusercontent.com/huggingface/datasets/1.7.0/datasets/story_cloze/story_cloze.py.
The file is also not present on the master branch on github.
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/tk/.virtualenvs/promptsource/lib/python3.8/site-packages/streamlit/script_runner.py", line 354, in _run_script
    exec(code, module.__dict__)
  File "/home/tk/repos/promptsource/promptsource/promptsource.py", line 112, in <module>
    pool.map(get_infos, all_datasets)
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 364, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 771, in get
    raise self._value
FileNotFoundError: Couldn't find file locally at story_cloze/story_cloze.py, or remotely at https://raw.githubusercontent.com/huggingface/datasets/1.7.0/datasets/story_cloze/story_cloze.py.
The file is also not present on the master branch on github.
```

and

```bash
Traceback (most recent call last):
  File "/home/tk/.virtualenvs/promptsource/lib/python3.8/site-packages/streamlit/script_runner.py", line 354, in _run_script
    exec(code, module.__dict__)
  File "/home/tk/repos/promptsource/promptsource/promptsource.py", line 112, in <module>
    pool.map(get_infos, all_datasets)
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 364, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 771, in get
    raise self._value
FileNotFoundError: Couldn't find file locally at Zaid/Zaid.py, or remotely at https://raw.githubusercontent.com/huggingface/datasets/1.7.0/datasets/Zaid/Zaid.py.
The file is also not present on the master branch on github.
```

**Comment by stephenbach** [comment]
Related to both #498 and #503. Should be fixed when those are both resolved.

**Comment by stephenbach** [comment]
#498 is merged and #503 is closed. This should be fixed now.
